recipe/0102-Fixed-failure-on-baremetal.patch

```diff
@@ -1,11 +1,20 @@
+From 4537122017775895019f3286e9232a07a79b4227 Mon Sep 17 00:00:00 2001
+From: Deepali Chourasia <deepch23@in.ibm.com>
+Date: Thu, 19 Aug 2021 10:05:45 +0000
+Subject: [PATCH] fix build failure on baremetal
+
+---
+ setup.py | 6 +++++-
+ 1 file changed, 5 insertions(+), 1 deletion(-)
+
 diff --git a/setup.py b/setup.py
-index fd653fe..28cab6d 100644
+index 4cc3d069..7dec64ff 100644
 --- a/setup.py
 +++ b/setup.py
-@@ -182,10 +182,12 @@ def get_extensions():
+@@ -185,10 +185,12 @@ def get_extensions():
      define_macros = []
  
-     extra_compile_args = {}
+     extra_compile_args = {'cxx': []}
```

Comment:
I'm curious why this is needed.

Reply:
This change is not done by us. It is coming from upstream - https://github.com/pytorch/vision/blob/v0.10.0/setup.py#L187.

In v0.9.1 it was - https://github.com/pytorch/vision/blob/v0.9.1/setup.py#L184.